src/main/java/lottogame/LottoMain.java

```diff
@@ -13,12 +13,20 @@ public class LottoMain {
 
     public static void main(String[] args) {
         Money money = InputView.requestMoney();
+        ResultView.printLinkBreak();
         LottoGame lottoGame = new LottoGame();
-        List<Lottos> lottos = lottoGame.createLottos(money);
-        ResultView.printCountOfPurchaseLottos(lottos);
+        Number countOfLottos = lottoGame.calculateCountOfLottos(money);
+        Number manualCountOfLotto = InputView.requestCountOfManualLottos(countOfLottos);
+        ResultView.printLinkBreak();
+        List<List<Number>> manualInputNumbers = InputView.requestMultipleManualLottos(manualCountOfLotto);
```

Comment:
```suggestion
        List<Numbers> manualInputNumbers = InputView.requestMultipleManualLottos(manualCountOfLotto);
```

Numbers가 있으면 어떨까요? ㅎ

만약 이러한 객체지향 패턴에 익숙하지 않다면, 계속해서 의문이 드실꺼에요.
저는 처음에 객체지향 패턴을 할때 이런생각을 했었어요.
`"이렇게 계속 그냥 값만 포장하는 객체를 만드는게 맞을까?"` 이런 고민이요 😂 

그러면서 코드 다 버리고 다시 해보고 또 다시 해보고 하면서 점점 객체에게 메시지를 보내는것이 익숙해지고
익숙해지니까 책임에 적합한 설계를 어떻게 해야할지 느낌이 오더라구요.

계속해서 시도해보시면 좋을것같아요!
아주 잘하고 있습니다! 😄

Reply:
[f1fd518](https://github.com/next-step/java-lotto/pull/3753/commits/f1fd518c1cbca810ec9527048a50c25044ded771) 적용했습니다~

그리고 `toLottosNumbers` 같은 경우는 아무래도 `Numbers` 도메인의 책임에 어울리지 않아서 
[24ff1a0](https://github.com/next-step/java-lotto/pull/3753/commits/24ff1a060e4763e1c8cde9f916f36e27048b1dd3) 에서 수정했습니다~